apps/settings/js/panels/root/root.js

```diff
@@ -27,6 +28,7 @@ define(function(require) {
        */
       LazyLoader.load([
         'js/firefox_accounts/menu_loader.js',
+        'js/dsds_settings.js',
```

Comment:
remove the usage of DsdsSettings in root panel, make DsdsSettings could be lazyloaded later